src/Compilers/CSharp/Portable/Syntax/SyntaxExtensions.cs

```diff
@@ -30,6 +30,13 @@ internal static ArrowExpressionClauseSyntax GetExpressionBodySyntax(this CSharpS
                 case SyntaxKind.MethodDeclaration:
```

Comment:
This should probably feature all inheritors of `BaseMethodDeclarationSyntax` and retrieve the `ExpressionBody` that way.